Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_answer/translation.txt

```diff
@@ -1,5 +1,5 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=
-//「仕方ないです、ここは既成事実を作るしかないです！」=
+「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=“Mmm... after being seen like this, we're already in this together!”
```

Comment:
Doesn't match the one from [penetration/heroine_action/translation.txt](https://github.com/IllusionMods/KKS-Translation/pull/146/files#diff-9ed6e5c24dc6b6d8107cdcda32e46fbbccf6ede413e66dff5dc6a0408ea4c547), update this one to match